source/broadband/webpa_internal.c

```diff
@@ -1431,17 +1432,14 @@ WDMP_STATUS check_ethernet_wan_status()
     char *status = NULL;
     char isEthEnabled[64]={'\0'};
 #ifdef RDKB_BUILD
-    if(0 == syscfg_init())
+    if( 0 == syscfg_get( NULL, "eth_wan_enabled", isEthEnabled, sizeof(isEthEnabled)) && (isEthEnabled[0] != '\0' && strncmp(isEthEnabled, "true", strlen("true")) == 0))
     {
-        if( 0 == syscfg_get( NULL, "eth_wan_enabled", isEthEnabled, sizeof(isEthEnabled)) && (isEthEnabled[0] != '\0' && strncmp(isEthEnabled, "true", strlen("true")) == 0))
-        {
-            WalInfo("Ethernet WAN is enabled\n");
-            OnboardLog("Ethernet WAN is enabled\n");
-            eth_wan_status = TRUE;
-            return WDMP_SUCCESS;
-        }
+        WalInfo("Ethernet WAN is enabled\n");
+        OnboardLog("Ethernet WAN is enabled\n");
+        eth_wan_status = TRUE;
+        return WDMP_SUCCESS;
     }
-    else
+    return WDMP_FAILURE;
```

Comment:
As part of this change, it is returning failure immediately when ethAgent process is delayed to start/not running in prod rdkb scenarios. In existing code, if it is not enabled, it will wait until ethAgent to come up and proceed. Is this logic change intentional?

Cc : @hpatel006c

Reply:
Hi @sadhyama 
Suggested review comments are fixed.
Can you please review and merge?